conda-recipes/README.md

```diff
@@ -29,20 +29,21 @@ The top level travis.yml file (https://github.com/IBM/powerai/blob/master/.travi
 jobs:
   include:
 
-    # tensorflow-hub recipe CONDA_PY=36
```

Comment:
Just now, I realized that this file need not to be changed. tensorflow-hub was added here as an example. But I think there is even no harm even if I change it because both packages are same. But going forward I won't change this.